### PR TITLE
[T4A5][F09-B4] Tan Yong Sheng

### PR DESCRIPTION
**Pull request by e-tys**

### CODE REVIEW

**Comment by e-tys** [comment]
Ready for review

File: src/seedu/addressbook/data/tag/Tagging.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        return this.tag;
+    }
+    
+    public Person getPerson() {
+        return this.person;
+    }
+    
+    @Override
+    public String toString() {
+        return (this.action == Action.ADD) ? ADD_ACTION : DELETE_ACTION + " " + this.person.getName() + " " + this.tag;
```
Comment (chanjunweimy) [inline src/seedu/addressbook/data/tag/Tagging.java:41]: Maybe we can break this into several lines for better practice? :)

**Comment by chanjunweimy** [comment]
@e-tys Some comments added. Please close the PR after reading comments. Good job! :)

File: src/seedu/addressbook/data/tag/Tagging.java (hunk 1 above)
Comment (e-tys) [reply src/seedu/addressbook/data/tag/Tagging.java:41]: Noted. Next time I will break it up.

**Pull request closed without merging**